ui/actions/nuxeo-delete-blob-button.js

```diff
@@ -150,15 +155,67 @@ import '../nuxeo-button-styles.js';
       };
     }
 
+    execute(batchId, fileId) {
+      return this.$.nx.connect().then(this._send(batchId, fileId));
+    }
+
+    _send(batchId, fileId) {
+      let url = [this.$.nx.client._baseURL, 'api/v1/upload', this.index, batchId].join('/');
+      url = url.replace(/(^\/+)|([^:])\/\/+/g, '$2/');
+      url += `/`;
+      url += fileId;
+      const options = {
+        method: 'DELETE',
+        headers: { 'Content-Type': 'application/json' },
+        url,
+      };
+      return this.$.nx.request().then((request) =>
+        request
+          .execute(options)
+          .then(this._handleResponse.bind(this))
+          .catch(this._handleError.bind(this)),
+      );
+    }
```

Comment:
Instead of Implementing request call . Kindly use Nuxeo-Resource and Nuxeo Connection both